src/models/lstm/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.10)
+add_library(models_lstm STATIC
+        LSTM.cpp)
+add_library(NGen::models_lstm ALIAS models_lstm)
+target_include_directories(models_lstm PUBLIC
+        ${PROJECT_SOURCE_DIR}/include
+        ${PROJECT_SOURCE_DIR}/include/utilities
+        ${PROJECT_SOURCE_DIR}/models/lstm/include
+        )
+
+target_link_libraries(models_lstm PUBLIC
+        Boost::boost                # Headers-only Boost
+        )
+
+if(LSTM_TORCH_LIB_ACTIVE)
```

Comment:
If  Torch  isn't  active, is there any reason to do `add_library` or any of the other cmake  functions here?

Reply:
Good point. I changed it all to go within the if.